dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -1299,6 +1302,10 @@ synchronized String evaluateInterval() {
                     // do nothing
                 }
                 completedWork = threadPool.getCompletedTaskCount();
+                if (tc.isEventEnabled()) {
+                    Tr.event(tc, "\nExecutor hang detected - poolSize: " + poolSize + ", queueDepth: " + queueDepth +
+                                 ", cpuUtil: " + cpuUtil + ", processCpuUtil: " + processCpuUtil + ", systemCpuUtil: " + systemCpuUtil);
+                }
```

Comment:
Do you want some additional details in this event message?  targetPoolSize?  hangBufferPoolSize?